src/main/java/org/deri/cqels/engine/TimerVisitor.java

```diff
@@ -3,60 +3,61 @@
 public class TimerVisitor implements RouterVisitor{
 
 	public void visit(JoinRouter router) {
-		//System.out.println("Hello JoinRouter");
+		System.out.println("Hello JoinRouter");
```

Comment:
Because this class is used only for debugging. More than anywhere else, it is not used. When used contrary commented https://github.com/garayzuev/cqels/blob/master/src/main/java/org/deri/cqels/engine/HeuristicRoutingPolicy.java#L259